manifests/a/ashisoft/dff/8.1.0.1/ashisoft.dff.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: ashisoft.dff
+PackageVersion: 8.1.0.1
+PackageLocale: en-US
+Publisher: Ashisoft
+PackageName: Duplicate File Finder
+License: Proprietary
+Copyright: Ashisoft all rights reserved.
+ShortDescription: Duplicate File Finder Setup

```

```suggestion
ShortDescription: Duplicate File Finder is a powerful application that uses intelligent algorithms to identify all types of duplicate files.
```
Proper description of the application.